src/workflows-app/components/WorkflowInfoBox.test.ts

```diff
@@ -0,0 +1,92 @@
+import { act, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { h } from 'react-hyperscript-helpers';
+import { getLink } from 'src/libs/nav';
+import { makeCompleteDate } from 'src/libs/utils';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
+import { metadata as mockRunDetailsMetadata } from 'src/workflows-app/fixtures/test-workflow';
+
+import { WorkflowInfoBox, WorkflowInfoBoxProps } from './WorkflowInfoBox';
+
+const mockWorkflowInfoBoxProps: WorkflowInfoBoxProps = {
+  name: 'mock-workspace-name',
+  namespace: 'mock-workspace-namespace',
+  submissionId: 'mock-submission-uuid',
+  workflowId: 'mock-workflow-uuid',
+  workspaceId: 'mock-workspace-uuid',
+  showLogModal: jest.fn(),
+};
+
+jest.mock('src/libs/nav', () => ({
+  getCurrentUrl: jest.fn().mockReturnValue(new URL('https://app.terra.bio')),
+  getLink: jest.fn(),
+  goToPath: jest.fn(),
+}));
+
+jest.mock('src/workflows-app/utils/cromwell-metadata-utils', () => ({
+  ...jest.requireActual('src/workflows-app/utils/cromwell-metadata-utils'),
+  fetchMetadata: jest.fn().mockReturnValue(mockRunDetailsMetadata),
+}));
+
+describe('Workflow Info Box Rendering', () => {
+  it('should render the workflow timing section', async () => {
+    await act(() => render(h(WorkflowInfoBox, mockWorkflowInfoBoxProps)));
+
+    expect(screen.getByText('Workflow Timing:')).toBeInTheDocument();
```

Comment:
It's a stylistic choice, but since `getBy...` queries throw an error if the element is not found, `expect` is unnecessary here. However, an argument can be made that `expect(...).toBeInTheDocument();` more clearly communicates the intent.

```suggestion
    screen.getByText('Workflow Timing:');
```